src/components/landing/PrizeGallery.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import React, { useState } from "react";
+import cx from "classnames";
+
+import {
+  Button,
+  Heading,
+  P,
+} from "components/2020/uikit";
+
+type Props = {
+  prizes: {},
+};
+
+const PrizeGallery = ({
+  prizes,
+}: Props) => {
+  const [current, setCurrent] = useState(0);
+
+  const prev = () => { current === 0 ? setCurrent(prizes.length - 1) : setCurrent(current - 1); };
+  const next = () => { current === prizes.length - 1 ? setCurrent(0) : setCurrent(current + 1); };
```

Comment:
think this can be something like `setCurrent((current + 1) % prizes.length)`

Reply:
🤯